src/main/java/de/tsystems/mms/apm/performancesignature/model/CredProfilePair.java

```diff
@@ -70,12 +67,32 @@ public String getDisplayName() {
             return "";
         }
 
-        public ListBoxModel doFillCredentialsIdItems(@AncestorInPath final Project project) {
-            return new StandardListBoxModel()
-                    .withEmptySelection()
-                    .withMatching(instanceOf(UsernamePasswordCredentials.class),
-                            CredentialsProvider.lookupCredentials(
-                                    StandardUsernameCredentials.class, project, ACL.SYSTEM, Collections.<DomainRequirement>emptyList()));
+        public ListBoxModel doFillCredentialsIdItems(@AncestorInPath Jenkins context, @QueryParameter String credentialsId) {
```

Comment:
Why have you changed from `Project` to `Jenkins`. You probably want this to be `TopLevelItem` and then use `item instanceof Queue.Task ? Tasks.getDefaultAuthenticationOf(item) : ACL.SYSTEM` in place of `ACL.SYSTEM`